images/server/install-packages.sh

```diff
@@ -6,7 +6,6 @@ get_custom_repo() {
     url="$1"
     fname="$(basename "$url")"
     dest="/etc/yum.repos.d/${fname}"
-    dnf install --setopt=install_weak_deps=False -y curl
```

Comment:
Just like in #98 these script file changes will break the fedora build.  Either we need a 2nd script OR we need to generalize the script to support both centos and fedora. I prefer the latter (and am willing to make the changes).

Reply:
the latter has happened (in collaboration ...)